src/xrpld/rpc/handlers/LedgerEntry.cpp

```diff
@@ -36,820 +37,591 @@
 
 namespace ripple {
 
-static STArray
-parseAuthorizeCredentials(Json::Value const& jv)
+static Expected<uint256, Json::Value>
```

Comment:
The moving of the functions makes this change near to impossible to review.  Consider to make as low differences as possible. Could you please add the new functions to the end of the file and use forward declaration when you need them.

Reply:
I split that out into a separate commit